src/main/java/domain/numberscontainer/SixLottoNumbersDTO.java

```diff
@@ -0,0 +1,30 @@
+package domain.numberscontainer;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class SixLottoNumbersDTO {
```

Comment:
```suggestion
public class SixLottoNumbersDto {
```

저는 DTO의 네이밍을 위와 같이 하는데요!
만약 `HtmlXmlHttpUrlDto` 와 같은 축약 형태로 이루어진 이름을 가진 이름은 어떻게 만들 수 있을까요?

Reply:
아이코 약자라서 대문자를 쓰는 게 맞다고 생각했는데 HTMLXMLHTTPURLDTO모두 대문자가 되겠네요! 감사합니다 반영해서 수정해보겠습니다! :D